drivers/flash/flash_stm32_qspi.c

```diff
@@ -1173,6 +1317,16 @@ static struct flash_stm32_qspi_data flash_stm32_qspi_dev_data = {
 			.ClockMode = QSPI_CLOCK_MODE_0,
 			},
 	},
+	.qer_type = COND_CODE_1(DT_INST_NODE_HAS_PROP(0, quad_enable_requirements),
+			(_CONCAT(JESD216_DW15_QER_VAL_,
+				 DT_STRING_TOKEN(DT_DRV_INST(0),
+						       quad_enable_requirements))),
+			(JESD216_DW15_QER_VAL_S1B6)),
+	.qspi_write_cmd = COND_CODE_1(DT_INST_NODE_HAS_PROP(0, writeoc),
+			(_CONCAT(SPI_NOR_CMD_,
+				 DT_STRING_UPPER_TOKEN(DT_DRV_INST(0),
```

Comment:
```suggestion
				 DT_STRING_TOKEN(DT_DRV_INST(0),
```

Reply:
Also please introduce macros to increase readability

Reply:
Wasn't sure about the macro names. I hope they are unique enough as they relate to DT and certain properties.